Sources/KafkaTestUtils/TestRDKafkaClient+TopicCreation.swift

```diff
@@ -0,0 +1,152 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the swift-kafka-client open source project
+//
+// Copyright (c) 2023 Apple Inc. and the swift-kafka-client project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of swift-kafka-client project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Crdkafka
+@_spi(Internal) import Kafka
+import struct Foundation.UUID
+
+public extension TestRDKafkaClient {
```

Comment:
I am a bit unsure about this here. We want to have an admin client anyhow so maybe we should just start creating an AdminClient that we can use in our benchmarks to set everything up. I am not a big fan of having something here that reaches into the rd_kafka internals outside of our main module.